svelte/src/libs/translations/languages/Estonian language

```diff
@@ -0,0 +1,101 @@
+{
```

Comment:
@nondevspell thank you for contributing but the file type is wrong. it be a json file. ee.json

Reply:
Check it now please

Reply:
@nondevspell still wrong.

Reply:
@nondevspell please check as reference: https://github.com/pkgxdev/ossapp/pull/851

Reply:
maybe now?

Reply:
yeah but please fix the conflicts.

Reply:
try it know I think all is fixed

Reply:
the pr is still showing it has conflicts check below

Reply:
![image](https://github.com/pkgxdev/ossapp/assets/156613281/4f53a2c5-f5eb-4138-857b-efe795e3c161)
I solved conflicts but no changes

Reply:
can you try to edit my PR?

Reply:
Maybe I need to put ee.json before kr.json as in japanese reference